iina/Base.lproj/Localizable.strings

```diff
@@ -11,78 +11,78 @@
-"osd.sub_delay.later" = "Subtitle Delay: %.2fs Later";
-"osd.sub_delay.earlier" = "Subtitle Delay: %.2fs Earlier";
-"osd.subtitle_pos" = "Subtitle Position: %f";
-"osd.mute" = "Mute";
-"osd.unmute" = "Unmute";
-"osd.screenshot" = "Screenshot captured";
-"osd.abloop.a" = "AB-Loop: A";
-"osd.abloop.b" = "AB-Loop: B";
-"osd.abloop.clear" = "AB-Loop: Cleared";
-"osd.stop" = "Stopped";
-"osd.chapter" = "Chapter: %@";
-"osd.subtitle_scale" = "Subtitle Scale: %.2fx";
-"osd.add_to_playlist" = "Added %i files to playlist";
-"osd.clear_playlist" = "Cleared playlist";
-"osd.video_eq.contrast" = "Contrast: %i";
+"osd.audio_delay.nodelay" = "Délai Audio: Aucun Délai";
+"osd.audio_delay.later" = "Délai Audio: %.2fs de Retard";
+"osd.audio_delay.earlier" = "Délai Audio: %.2fs Earlier";
+"osd.sub_delay.nodelay" = "Délai Sous-Titres: Aucun Délai";
+"osd.sub_delay.later" = "Délai de Sous-Titres: %.2fs de Retard";
+"osd.sub_delay.earlier" = "Délai de Sous-Titres: %.2fs de Plus Tôt";
+"osd.subtitle_pos" = "Position de Sous-Titres: %f";
+"osd.mute" = "Couper le Son";
+"osd.unmute" = "Reprendre le Son";
+"osd.screenshot" = "Écran Capturé";
+"osd.abloop.a" = "Boucle-AB: A";
+"osd.abloop.b" = "Boucle-AB: B";
+"osd.abloop.clear" = "Boucle-AB: Frayée";
+"osd.stop" = "Arreté";
+"osd.chapter" = "Chapitre: %@";
+"osd.subtitle_scale" = "Échelle de Sous-Titres: %.2fx";
+"osd.add_to_playlist" = "%i Fichiers ajoutés a la liste de lecture";
+"osd.clear_playlist" = "Liste de lecture Frayée";
+"osd.video_eq.contrast" = "Contraste: %i";
 "osd.video_eq.gamma" = "Gamma: %i";
-"osd.video_eq.hue" = "Hue: %i";
+"osd.video_eq.hue" = "Teint: %i";
 "osd.video_eq.saturation" = "Saturation: %i";
-"osd.video_eq.brightness" = "Brightness: %i";
+"osd.video_eq.brightness" = "Luminosité: %i";
 
-"update.title_update_found" = "Update Found";
-"update.title_no_update" = "No Update";
-"update.title_error" = "Error";
-"update.update_found" = "New version detected! IINA is still in alpha, so you are recommended to stay with the latest version. Please download from website or github:";
-"update.no_update" = "You are running the latest version.";
-"update.check_failed" = "Failed to check updates. You are probably running a too old version. Please check the website or github.";
-"update.new_version" = "New version:";
-"update.dl_from_website" = "Go to Website";
-"update.dl_from_github" = "Go to Github";
+"update.title_update_found" = "Mise à jour Trouvée";
+"update.title_no_update" = "Aucune Mise à jour Trouvée";
+"update.title_error" = "Érreur";
+"update.update_found" = "Mise à jour trouvée! IINA est toujours en alpha, alors il est recommendé d'avoir la plus récente version. Les fichiers sont disponible sur le site web IINA ou sur GitHub:";
+"update.no_update" = "Vous avez la version la plus récente.";
+"update.check_failed" = "Echec de trouvaille de mise à jour. La version courrante est trop vieille. Les plus récent fichiers sont disponible sur le site web IINA ou sur GitHub.";
```

Comment:
"Impossible de vérifier les mises à jour. Vous avez probablement une version trop ancienne. Veuillez consulter le site internet ou Github" ?

"Trouvaille" here is a bit weird, and there's a little mistake: "courrante" should be "courante"; you also forgot the plural in "Les plus récent fichiers sont disponible" ("Les fichiers les plus récents sont disponibles"